src/components/SelectStatisticsMenu/SelectStatisticsMenu.jsx

```diff
@@ -0,0 +1,74 @@
+import React, {useEffect, useState} from "react";
+import styles from "../SelectStatisticsMenu/SelectStatisticsMenu.module.css"
+
+export const SelectStatisticsMenu = ({ placeholder, onClick, type }) => {
+    const dane = type
+    const fakeFetchData = () => {
+        const months = [
+            {'data': 'January'},
+            {'data': 'February'},
+            {'data': 'March'},
+            {'data': 'April'},
+            {'data': 'May'},
+            {'data': 'June'},
+            {'data': 'July'},
+            {'data': 'August'},
+            {'data': 'September'},
+            {'data': 'October'},
+            {'data': 'November'},
+            {'data': 'December'}]
+        const years = [
+            {'data': '2023'},
+            {'data': '2022'},
+            {'data': '2021'},
+            {'data': '2020'}]
```

Comment:
potrzebne będzie tłumaczenie że gdy jest 01 to jest wtedy January itd
generalnie prosiłem Mariusza żeby format otrzymywanych danych co do miesięcy i lat wyglądał tak:

```
[
{year: 2023,
months: [01,02,03,04]
},
{year:2022,
months: [11,12]
}
]
```
Wtedy baza danych by sprawdzała jakie miesiące są